doc/performance-investigations/0012-increase-machine-type-core-count/summary.md

```diff
@@ -0,0 +1,49 @@
+# Investigate Machine Type Architecture Performance
+
+EQ Runner currently uses a machine type of `custom-4-4096` (4 vCPU cores, 4096MB of RAM). This investigation looked at the impact of machine type on the performance of the runner application, focusing on the number of CPU cores as Runner is primarily a CPU bound application with limited network IO to Datastore and Redis.
+
+The investigation aims to determine whether latencies as measured by the Locsut client improve when using larger machines but utilizing roughly the same percentage of CPU.
+
+## Benchmark profile
+
+| Option                 | Value                        |
+| ---------------------- | ---------------------------- |
+| Requests file          | census_household_gb_eng.json |
+| Run time               | 20m                          |
+| User wait time minimum | 1                            |
+| User wait time maximum | 2                            |
+| Clients                | 256                          |
+| Number of workers      | 7                            |
+| Number of threads      | 7                            |
+
+This test replicates the current settings for the daily test. Note that the baseline test ran with 64 clients as per the daily test.
+
+## Results
+
+| Cores | Pods | Nodes | GET Average | POST Average | All Request Average | Total Requests |
+| ----- | ---- | ----- | ----------- | ------------ | ------------------- | -------------- |
+| 4     | 4    | 4     | 572         | 609          | 590                 | 403,890        |
+| 8     | 4    | 2     | 542         | 579          | 560                 | 404,535        |
+| 8     | 4    | 2     | 591         | 601          | 596                 | 402,884        |
+| 16    | 4    | 1     | 620         | 651          | 636                 | 138,195        |
+| 16    | 4    | 1     | 579         | 610          | 594                 | 139,156        |
```

Comment:
What is different about these tests?

Reply:
Copy paste error. The second 16 4 1 test Should have been 16 5 1

Reply:
The total requests still looks incorrect?

Reply:
Are you viewing an old version of the file? Should be 379,246 and 394,487

Reply:
Looks like it 🤦